src/pages/experiments/[experimentId]/data-processing/StepsIndicator.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const StepsIndicator = (props) => {
+  const { allSteps, completedSteps, currentStep } = props;
+  let increment = 0;
+
+  return (
+    <>
+      {allSteps.map((step) => {
+        let color = '#1890ff';
+        if (increment === currentStep) {
+          color = '#cf1322';
+        } else if (increment > completedSteps - 1) {
+          color = '#d9d9d9';
```

Comment:
why do we have 3 different colours used here when in practice I only see 2 different colours on the UI?

Reply:
also, it will be more readable if you name the colours differently - like `currentStepColor` and `completedStepColor` and then have `color = currentStepColor` or `color = completedStepColor`?

Reply:
So 3 different colors - gray for not-completed steps, blue for completed, red for the step you are currently in.

Reply:
ook, makes sense